lib/handlers/listing.js

```diff
@@ -44,19 +44,28 @@ function getListingsByUserID(user_id, callback) {
 }
 exports.getListingsByUserID = getListingsByUserID;
 
-function getListings(listing_type, callback) {
+function getListings(listing_type, pageNum, callback) {
     var params = {
         'integrity.visible': true,
         'integrity.flag': {'$ne':true }
     };
     if (listing_type != null) {
         params['type'] = listing_type;
     }
-    listing.model
-        .find(params)
-        .sort({lastRefreshed: 'desc'})
-        .limit(Consts.MAX_LISTING_RESULTS)
-        .exec(callback);
+    if (listing_type == 'SubletListing'){
```

Comment:
We can’t really paginate sublets because then the map becomes useless. We can come up with another solution for that but for now I think we shouldn’t block the other stuff since the solution for sublets is more involved. 
> On Sep 10, 2016, at 1:23 PM, Brian Cefali <notifications@github.com> wrote:
> 
> In lib/handlers/listing.js <https://github.com/TuftsTusk/ringmaster/pull/18#discussion_r78280683>:
> 
> >      var params = {
> >          'integrity.visible': true,
> >          'integrity.flag': {'$ne':true }
> >      };
> >      if (listing_type != null) {
> >          params['type'] = listing_type;
> >      }
> > -    listing.model
> > -        .find(params)
> > -        .sort({lastRefreshed: 'desc'})
> > -        .limit(Consts.MAX_LISTING_RESULTS)
> > -        .exec(callback);
> > +    if (listing_type == 'SubletListing'){
> Why aren't we paginating sublet listings?
> 
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub <https://github.com/TuftsTusk/ringmaster/pull/18/files/a10bf340a47edfc004aa498629e4c1855a685aba#r78280683>, or mute the thread <https://github.com/notifications/unsubscribe-auth/ACtOnceA7AyW6KbGaEIlTPJlrM0e7BkNks5qouejgaJpZM4J4TmX>.
>